diff -ur 000/pidgin-2.10.12/libpurple/win32/global.mak 001/pidgin-2.10.12/libpurple/win32/global.mak
--- 000/pidgin-2.10.12/libpurple/win32/global.mak	2015-12-31 21:19:41.000000000 -0200
+++ 001/pidgin-2.10.12/libpurple/win32/global.mak	2016-01-08 16:34:41.535120000 -0200
@@ -16,7 +16,7 @@
 GTK_BIN ?= $(GTK_TOP)/bin
 LIBXML2_TOP ?= $(WIN32_DEV_TOP)/libxml2-2.9.2_daa1
 MEANWHILE_TOP ?= $(WIN32_DEV_TOP)/meanwhile-1.0.2_daa3
-NSS_TOP ?= $(WIN32_DEV_TOP)/nss-3.17.3-nspr-4.10.7
+NSS_TOP ?= $(WIN32_DEV_TOP)/nss-3.20.1-nspr-4.10.10
 PERL_LIB_TOP ?= $(WIN32_DEV_TOP)/perl-5.20.1.1
 SILC_TOOLKIT ?= $(WIN32_DEV_TOP)/silc-toolkit-1.1.12
 GSTREAMER_TOP ?= $(WIN32_DEV_TOP)/gstreamer-0.10.13
diff -ur 000/pidgin-2.10.12/pidgin/win32/nsis/generate_gtk_zip.sh 001/pidgin-2.10.12/pidgin/win32/nsis/generate_gtk_zip.sh
--- 000/pidgin-2.10.12/pidgin/win32/nsis/generate_gtk_zip.sh	2015-12-31 21:19:42.000000000 -0200
+++ 001/pidgin-2.10.12/pidgin/win32/nsis/generate_gtk_zip.sh	2016-01-08 08:12:50.641396600 -0200
@@ -18,7 +18,7 @@
 
 #This needs to be changed every time there is any sort of change.
 BUNDLE_VERSION=2.16.6.3
-BUNDLE_SHA1SUM=e1b1ec8d2159fa98b2a9f516dbfe745bf7a22169
+BUNDLE_SHA1SUM=071e95e32eb09361fcec1e62feb82bf80b41ec84
 ZIP_FILE="$PIDGIN_BASE/pidgin/win32/nsis/gtk-runtime-$BUNDLE_VERSION.zip"
 SRC_ZIP_FILE="$PIDGIN_BASE/pidgin/win32/nsis/gtk-runtime-$BUNDLE_VERSION-src.zip"
 
